lib/callbacks/DebugSource.cpp

```diff
@@ -35,13 +35,15 @@ namespace MAT_NS_BEGIN {
     /// <summary>Microsoft Telemetry SDK invokes this method to dispatch event to client callback</summary>
     bool DebugEventSource::DispatchEvent(DebugEvent evt)
     {
-        seq++;
-        evt.seq = seq;
-        evt.ts = PAL::getUtcSystemTime();
+        int64_t timestamp = PAL::getUtcSystemTime();
```

Comment:
`evt.ts = PAL::getUtcSystemTime();` needn't be inside critical region?

Reply:
Updated, thanks!